Lib/Inc/IComChannel.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * IComChannel.h
+ *
+ *  Created on: 16 Feb 2022
+ *      Author: salavat.magazov
+ */
+
+#ifndef INC_ICOMCHANNEL_H_
```

Comment:
Interesting. This was autogenerated, so not sure whey it's not matching the class name exactly. These are the include gateways though, so they are not subject to the rule about file name and class name matching. They are there to prevent recursive #include.